OpenRA.Mods.Common/Scripting/Media.cs

```diff
@@ -60,27 +59,11 @@ public static void PlayFMVFullscreen(World w, string movie, Action onComplete)
 			});
 		}
 
-		public static void PlayFMVInRadar(IVideo movie, Action onComplete)
+		public static void PlayFMVInRadar(string videoFileName, Action onComplete)
 		{
 			var player = Ui.Root.Get<VideoPlayerWidget>("PLAYER");
-			player.Open(movie);
-
-			player.PlayThen(() =>
-			{
-				onComplete();
-				player.CloseVideo();
-			});
-		}
-
-		public static void StopFMVInRadar()
-		{
-			var player = Ui.Root.Get<VideoPlayerWidget>("PLAYER");
-			player.Stop();
-		}
-
-		public static IVideo LoadVideo(Stream s)
-		{
-			return VideoLoader.GetVideo(s, true, Game.ModData.VideoLoaders);
+			player.DrawOverlay = false;
```

Comment:
I still don't understand this. It's never set to true, why is it here? Why is it needed? An explanation should be added to the variable

Reply:
The default value is `true`: https://github.com/OpenRA/OpenRA/blob/5bf7fe852c8f122e3a3e0a1eb72d66d7429675c3/OpenRA.Mods.Common/Widgets/VideoPlayerWidget.cs#L24

I guess there was another crash hiding behind the `.BeginInvoke()` one, so I added that line on a hunch (which paid off). Investigating it further now, I guess the issue is related to this:
![video_played_widget_draw_overlay](https://user-images.githubusercontent.com/7137365/217160511-5d5fc81b-e913-4a0c-9c2a-a91cf6ad9b0f.gif)
I have a better fix that I will push tonight after work.